coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -115,3 +140,23 @@ def get_ignore_comment(self, origin, language):
                         language=language))
                 ignore_comment = None
         return ignore_comment
+
+    def read_ignore_comments(self, original_line, comment_delimiter):
```

Comment:
this method is not about the action.  It is a generic algorithm which should be part of the relevant module which does ignore handling.

Reply:
I placed it here as it is explicitly for ignores added by coala only. I felt this is the most relevant file. Else please direct me to the file.

Reply:
No it isnt.  This is *action* code.  Any function which doesnt relate specifically to doing the action should be located elsewhere.  Find the modules which have code related to detecting, loading and processing ignore comments.